lib/views/pages/events/event_card_widget.dart

```diff
@@ -105,24 +105,28 @@ Widget popUpMenue(
               style: const TextStyle(color: Colors.black),
             ),
           )),
-      const PopupMenuItem<int>(
-          value: 3,
-          child: const ListTile(
-            leading: const Icon(Icons.edit, color: Colors.grey),
-            title: const Text(
-              'Edit this event',
-              style: const TextStyle(color: Colors.black),
-            ),
-          )),
-      const PopupMenuItem<int>(
-          value: 4,
-          child: const ListTile(
-            leading: const Icon(Icons.delete, color: Colors.grey),
-            title: const Text(
-              'Delete This Event',
-              style: const TextStyle(color: Colors.black),
-            ),
-          ))
+      event.creator.id == model.userID
```

Comment:
recommend using this syntax, you get to cover both buttons without it looking awkward

```dart
if(event.creator.id == model.userID)... [
//Admin Edit Widget here,
//Admin Delete Widget here,
],
```

Reply:
Thanks 
I learned new thing today :)